wrapper/src/main/java/software/amazon/jdbc/ds/AwsWrapperDataSource.java

```diff
@@ -99,6 +99,11 @@ public Connection getConnection(String username, String password) throws SQLExce
       // If the url is set explicitly through setJdbcUrl or the connection properties.
       if (!isNullOrEmpty(this.jdbcUrl)
           || (!isNullOrEmpty(this.urlPropertyName) && !isNullOrEmpty(props.getProperty(this.urlPropertyName)))) {
+        if (!isNullOrEmpty(this.jdbcUrl)) {
```

Comment:
We can just use `StringUtils.isNullOrEmpty` instead, and remove this local `isNullOrEmpty` implementation.